Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
Instead of using a marker file, which can get easily out of sync, I'd like to use each @colon@ file as dependency here. @matthiasr we do something similar in a couple of SoundCloud projects, do you have some time to use your make skills to apply that here?

I'll try to port it this weekend otherwise.